python/spec/dependabot/python/update_checker/index_finder_spec.rb

```diff
@@ -124,6 +124,19 @@
         end
       end
 
+      context "set pypi explicitly in a pyproject.toml" do
+        let(:pyproject_fixture_name) { "pypi_explicit.toml" }
+        let(:dependency_files) { [pyproject] }
+
+        it { is_expected.to eq(["https://pypi.org/pypi/"]) }
+
+        context "that is unparseable" do
+          let(:pyproject_fixture_name) { "unparseable.toml" }
+
+          it { is_expected.to eq(["https://pypi.org/simple/"]) }
```

Comment:
I wouldn't expect the unparseable example to be able to return anything here? 🤔

Reply:
Whoops. This is the first time a cloned this repo, and also my first PR. It seems your keen eyes caught on to the fact that I copy-pasted the test from above 😉 

Good catch! I will remove it